pages/dkp/kommander/2.2/licensing/enterprise/index.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: layout.pug
+navigationTitle: DKP Enterprise
+title: DKP Enterprise
+excerpt: A section containing all of the features available to users with a DKP Enterprise license.
+menuWeight: 5
+beta: false
+---
+
+## DKP Enterprise overview
+
+![DKP Enterprise Diagram](../img/dkpenterprisediagram.png)
+
+DKP Enterprise is a multi-cluster lifecycle management Kubernetes solution centered around a FIPS compliant management cluster that manages multiple attached or managed Kubernetes clusters through a centralized management dashboard that gives you a single point of observability and control throughout all of your attached or managed clusters. The DKP Enterprise license gives the user access to the entire Konvoy cluster environment, the Kommander UI dashboard that deploys platform and catalog applications as well as multi-cluster management, and comprehensive compatibility with our full range of infrastructure deployment options.
+
+## Compatible infrastructure
+
+DKP Enterprise operates across D2iQ's entire range of cloud, on-premise, edge, and air-gapped infrastructures and has support for various OSs, including immutable OSs.
+
+Deployment options for DKP Enterprise include:
+
+* Bare metal servers
+* VMware vSphere   
+
+Cloud deployment options for DKP Enterprise include:
+
+* Amazon Web Services (AWS)
+* Amazon Elastic Kubernetes Service (EKS)
+* Azure Kubernetes Service (AKS)
+* Google Kubernetes Engine (GKE)
+* Lightweight Kubernetes (K3s)
+* Microsoft Azure
+* Google Cloud Platform (GCP)
+
+For the basics on standing up a DKP Enterprise cluster in one of the previously listed environments of your choice, see [Choose Infrastructure][choose-infrastructure].  
+
+## Platform applications
+
+Applications can be deployed in any Kubernetes-managed or self-attached clusters, giving you complete flexibility to operate across cloud, on-premise, edge, and air-gapped scenarios. DKP Enterprise users can use the Kommander UI to customize which platform applications to deploy to the cluster in a given workspace. For a list of available platform applications that are included with DKP Enterprise, see [Workspace Platform Applications][workspaceplatform].
```

Comment:
I don't think that's the correct link.   'Platform applications' are installed by Kommander on the Management cluster.   'Workspace Platform applications' are a subset of the platform applications which can be optionally installed on an attached or managed cluster.   I am not sure where to link for a concise list of the platform applications.

Reply:
Just going to delete link sentence. If we dont have supporting platform apps docs besides whats listed then it shouldnt be linked.